api/swagger_v2.yaml

```diff
@@ -0,0 +1,1138 @@
+                $ref: '#/components/schemas/Problem7807'
+        '500':
+          description: >-
+            Internal Server Error.  Unexpected condition encountered when
+            processing the request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+  /subscriptions/{xname}/agents:
+    parameters:
+      - in: path
+        name: xname
+        required: true
+        schema:
+          $ref: '#/components/schemas/XName.1.0.0'
+    delete:
+      tags:
+        - subscriptions
+      summary: Delete all state change notification subscriptions for a component
+      description: >-
+        Delete all state change notification subscriptions for a component.
+      operationId: doSubscriptionDeleteXName
+      responses:
+        '204':
+          description: Success.
+        '401':
+          description: >-
+            Unauthorized. RBAC prevented operation from executing, or
+            authentication token has expired.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '404':
+          description: Does Not Exist.  Endpoint not available.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '405':
+          description: >-
+            Operation Not Permitted.  Only DELETE operations are allowed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '500':
+          description: >-
+            Internal Server Error.  Unexpected condition encountered when
+            processing the request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+  /subscriptions/{xname}/agents/{agent}:
+    parameters:
+      - in: path
+        name: xname
```

Comment:
tell me what xname is here... the xname I want to subscribe against? or MY xname?

Reply:
add a description

Reply:
Fixed.